x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertGenUtils.java

```diff
@@ -256,7 +257,7 @@ static PKCS10CertificationRequest generateCSR(KeyPair keyPair, X500Principal pri
      * Gets a random serial for a certificate that is generated from a {@link SecureRandom}
      */
     public static BigInteger getSerial() {
-        SecureRandom random = new SecureRandom();
+        SecureRandom random = Randomness.createSecure();
```

Comment:
Maybe it is useful in test repro to have the same keys and certs (but the validity date changes for certs).